bfillings/vsearch.py

```diff
@@ -222,6 +222,43 @@ def getHelp(self):
         return help_str
 
 
+def parse_uc_to_clusters(uc_filepath, seed_id_field=8):
```

Comment:
Just curious - there isn't already a function to parse the UC files in the uclust or the usearch app controller? I think it will help to reduce code duplication